assets/js/shared/context/product-data-context.js

```diff
@@ -28,7 +28,10 @@ const defaultProductData = {
 		price: '0',
 		regular_price: '0',
 		sale_price: '0',
-		price_range: null,
+		price_range: {
+			min_amount: '0',
+			max_amount: '0',
+		},
```

Comment:
This part had been changed to prevent a TS error in the following section:
- https://github.com/woocommerce/woocommerce-blocks/blob/8e8b4a003d0d95c98b0cb388d034a0b73709f1e2/assets/js/atomic/blocks/product-elements/price/block.tsx#L68-L69

Reply:
I checked the corresponding API response via a `GET` request to `https://store.test/wp-json/wc/store/v1/products/7` and received the following response:

```json
...
"prices": {
    "price": "4620",
    "regular_price": "4950",
    "sale_price": "4620",
    "price_range": {
        "min_amount": "4620",
        "max_amount": "4950"
    },
    "currency_code": "EUR",
    "currency_symbol": "€",
    "currency_minor_unit": 2,
    "currency_decimal_separator": ".",
    "currency_thousand_separator": ",",
    "currency_prefix": "€ ",
    "currency_suffix": ""
},
...
```

Reply:
@mikejolley Given that you implemented the initial code in #2871, would you mind double-checking that changing
```js
price_range: null,
```
to
```js
price_range: {
    min_amount: '0',
    max_amount: '0',
},
```
does not introduce a breaking change?

Reply:
`price_range` should be null unless the product is variable. The type is `price_range: null | { min_amount: string; max_amount: string };`

Reply:
Thanks for clarifying this, @mikejolley. Given that [assets/js/shared/context/product-data-context.js](https://github.com/woocommerce/woocommerce-blocks/pull/7683/files/8e8b4a003d0d95c98b0cb388d034a0b73709f1e2#diff-b7b79e54bd9ea5d196c77bfe00cf06b6372f49b53689508a19453408ffa1674d) hasn't been converted yet, I cannot apply TS there. Therefore, I defined the following interface https://github.com/woocommerce/woocommerce-blocks/blob/2f699ca8723dc3b3d5f4f4f23e48188590c5df58/assets/js/atomic/blocks/product-elements/price/block.tsx#L22-L34